docs/configuring.rst

```diff
@@ -97,7 +97,7 @@ Miscellaneous options
 .. confval:: analytics_id
 
     If specified, Google Analytics' javascript is included in your pages.
```

Comment:
We should add docs here about the particular implementation we use.

Reply:
Added a link to `gtag.js` in the docs